library/std/src/io/util.rs

```diff
@@ -58,6 +60,26 @@ impl BufRead for Empty {
     fn consume(&mut self, _n: usize) {}
 }
 
+#[stable(feature = "empty_seek", since = "1.50.0")]
```

Comment:
Just dropping a comment here that this `since` field will need to be bumped if the FCP completes.